data/workshops/2022.md

```diff
@@ -40,28 +118,11 @@ program_committee:
     affiliation: FAMAF, UNC and Manas.Tech, Argentina
 ---
 
-## Scope
+OCaml Workshop 2022 was co-located with ICFP 2022, in Ljubljana, Slovenia
 
-Presentations and discussions focus on the OCaml programming language and its community. We aim to solicit talks on all aspects related to improving the use or development of the language and its programming environment, including, for example (but not limited to):
+ACM Sigplan ICFP page: [OCaml Workskop 2022](https://icfp22.sigplan.org/home/ocaml-2022)
 
- - compiler developments, new backends, runtime and architectures
- - practical type system improvements, such as GADTs, first-class modules, generic programming, or dependent types
- - new library or application releases, and their design rationales
- - tools and infrastructure services, and their enhancements
- - prominent industrial or experimental uses of OCaml, or deployments in unusual situations.
+The OCaml Users and Developers Workshop brings together the OCaml community, including users of OCaml in industry, academia, hobbyists, and the free software community.
 
-## Presentations
+Videos will be made available soon.
 
-The workshop is an informal meeting with no formal proceedings. The presentation material will be available online from the workshop homepage. The presentations may be recorded and made available at a later date.
-
-The main presentation format is a workshop talk, traditionally around 20 minutes in length, plus question time, but we also have a poster session during the workshop - this allows to present more diverse work, and gives time for discussion. The program committee will decide which presentations should be delivered as posters or talks.
-
-## Submission
-
-To submit a presentation, please register a description of the talk (about 2 pages long) at [https://ocaml2022.hotcrp.com/](https://ocaml2022.hotcrp.com/) providing a clear statement of what will be provided by the presentation: the problems that are addressed, the solutions or methods that are proposed.
-
-LaTeX-produced PDFs are a common and welcome submission format. For accessibility purposes, we ask PDF submitters to also provide the sources of their submission in a textual format, such as .tex sources. Reviewers may read either the submitted PDF or the text version.
-
-## ML family workshop
```

Comment:
ML Family Workshop
(capitalized)

Reply:
Many thanks for all your comments. This text was deleted